overlore/townhall/service.py

```diff
@@ -5,12 +5,11 @@
 import signal
 from typing import Any
 
+import sqlean
```

Comment:
temporarily connecting/querying world db in the entry point for debug convenience, can remove and add something simple that can be iterated on in a subsequent PR